library/common/internal/plugin_handler.cc

```diff
@@ -94,4 +84,14 @@ void PluginHandler::SetMessageHandler(const std::string &channel,
   handlers_[channel] = std::move(handler);
 }
 
+// PluginRegistrar:
+
```

Comment:
Remove extra line?

Reply:
It's a heading for both of the following methods, so I separated it so that it doesn't appear to be associated only with AddPlugin.